### PR TITLE
feat: new lead-in price calculation

### LINKED ISSUE #189: Change the pricing curve based on the new code in the broker pallet
There has been a major change in the implementation of the broker pallet - see [link](https://github.com/paritytech/polkadot-sdk/blob/eca1052ea1eddeede91da8f9f7452ea8b57e7942/substrate/frame/broker/src/adapt_price.rs#L109-L117)

this change hasn't been yet implemented yet on Kusama, but should be already active on Rococo. You can help yourself with looking at the following graph while implementing it.
Screenshot 2024-06-13 at 20 37 50

**Note**
Once you take on this PR paste a comment with your hand -> ✋ so that people know that someone is working on this issue.

### PR DESCRIPTION
**Pull request by tien**
Resolves #189

### CODE REVIEW

**Comment by poppyseedDev** [comment]
@tien Code looks good! Let's just do a mini `NOTE` message (maybe bellow the graph) that there is a new pricing model as suggested by: https://grillapp.net/12935/agile-coretime-pricing-explained-166522

**Comment by tien** [comment]
K, I've added a mini footnote explaining the new pricing model, not a designer though, so do let me know a better placement for it :p

File: src/app/[network]/(App)/bulkcore1/CoreUtilisation.tsx [hunk 1]
```diff
@@ -64,6 +74,18 @@ const CoreUtilisation: React.FC = () => {
         : [],
       xLabel: 'Block Number',
       yLabel: `Price - ${network_list[network].tokenSymbol}`,
+      footnote: (
+        <span>
+          Pricing curve indicative of the{' '}
+          <a
+            className="hover:underline"
```
Comment (poppyseedDev) [inline src/app/[network]/(App)/bulkcore1/CoreUtilisation.tsx:81]: ``

Comment (poppyseedDev) [inline src/app/[network]/(App)/bulkcore1/CoreUtilisation.tsx:81]:
```suggestion
            className="underline hover:font-bold"
```

**Comment by poppyseedDev** [comment]
Nice! Let's merge and hope all goes well in the next sale! 👍

**Pull request merged**